src/snowflake/connector/connection.py

```diff
@@ -226,6 +226,10 @@ def DefaultConverterClass() -> type:
         True,
         bool,
     ),  # Enable sending retryReason in response header for query-requests
+    "disable_console_login": (
+        False,
```

Comment:
Default should be true.

Reply:
```suggestion
        True,
```

Reply:
How about a positive parameter that is set to True by default instead?

Reply:
The parameter used by JDBC is `disable_console_login` and True by default. Let's be consistent if no strong opinion otherwise?